src/ign_TEST.cc

```diff
@@ -37,6 +37,10 @@ static const std::string kIgnModelCommand(
 /////////////////////////////////////////////////
 std::string customExecStr(std::string _cmd)
 {
+  // Augment the system plugin path.
+  gz::common::setenv("IGN_GAZEBO_SYSTEM_PLUGIN_PATH",
```

Comment:
`IGN_GAZEBO_SYSTEM_PLUGIN_PATH` or `GZ_GAZEBO_SYSTEM_PLUGIN_PATH`

Reply:
This is targeting citadel, so `IGN_GAZEBO_SYSTEM_PLUGIN_PATH` is correct in this case.